src/System.Text.Json/docs/writable_json_dom_spec.md

```diff
@@ -164,7 +184,11 @@ string jsonString = @"
 }";
 
 JsonObject employees = JsonNode.Parse(jsonString) as JsonObject;
-employees.Add(EmployeesDatabase.GetNextEmployee());
+
+var newEmployee = new JsonObject({"name", "Bob"});
+var nextId = employees.Count + 1;
```

Comment:
Don't use var here. Also, I don't think we have `Count` on `JsonObject`.

Reply:
Right, it would need to be `employees.Values.Count`

Reply:
Or `PropertyNames` - it would be more precise.